enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -134,29 +145,29 @@ reconciled by two separate prometheus operators.
 Both prometheus operators are under supervision of cluster monitoring operator.
 This allows a separation of concerns in terms of upgradability, security settings, and scalability.
 The cluster monitoring Prometheus Operator continues to be deployed in the `openshift-monitoring` namespace.
-The user workload monitoring Prometheus Operator will be deployed in the `openshift-user-monitoring` namespace.
+The user workload monitoring Prometheus Operator is deployed in the `openshift-user-workload-monitoring` namespace.
 
 Currently, cluster monitoring operator scans all namespaces filtering `openshift` namespaces labeled with
 `openshift.io/cluster-monitoring: "true"` and takes this set of namespaces as the _allowed list_ of reconcilable
```

Comment:
So it's an AND today.  Can we instead honor just the label since users are not allowed to label their own namespaces?  this would free us from having to give away `openshift-` namespaces to every integration that thinks they should be included as a system feature.

Reply:
I think we should change the language here (sorry, as i am not a native speaker) but in fact we _do_ have the requested logic in place already: https://github.com/openshift/cluster-monitoring-operator/blob/e1b5cf1917133907e3a74e53db3b637e48d384a5/pkg/client/client.go#L248-L262

Reply:
There's a slight addition to that: the alerting setup is limited to those namespaces though, so while ServiceMonitors are selected if the namespace is non `openshift-` prefixed the setup is in a somewhat inconsistent state.

Reply:
Ah yes, indeed as per https://github.com/openshift/cluster-monitoring-operator/blob/d531ecb19943f445a3b8ed46d593dec4fd05674a/jsonnet/main.jsonnet#L132

Reply:
Created https://issues.redhat.com/browse/MON-901 epic to track this requirement and prioritization, cc @sichvoge.